<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jzys_widget</class>
 <widget class="QWidget" name="jzys_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>565</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>679</width>
    <height>565</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>679</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>21</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>21</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>21</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>73</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>73</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>73</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>125</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>125</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>125</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>177</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>141</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_ykh">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_dian">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_dengyu">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>＝</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_chu">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_jian">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>－</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_zkh">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton_jia">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>＋</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_cheng">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_quanqing">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>全清</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_tuige">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>退格</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
